lib/elixir/lib/task/supervisor.ex

```diff
@@ -67,6 +67,7 @@ defmodule Task.Supervisor do
   def async(supervisor, module, fun, args) do
     args = [self, :link, get_info(self), {module, fun, args}]
     {:ok, pid} = Supervisor.start_child(supervisor, args)
+    Process.link(pid)
```

Comment:
Curious. Why do we need this one? In case it crashed before establishing the link? Could that even happen (supervisor crashed)?